cli/cmd/install.go

```diff
@@ -85,7 +85,17 @@ func render(config installConfig, w io.Writer) error {
 	if err != nil {
 		return err
 	}
-	return InjectYAML(buf, w, conduitVersion)
+
+	proxyConfig := &proxyConfig{
+		version:         conduitVersion,
+		logLevel:        proxyLogLevel,
+		apiPort:         proxyAPIPort,
+		controlPort:     proxyControlPort,
+		outboundPort:    outboundPort,
+		inboundPort:     inboundPort,
+		eventBufferSize: 0,
```

Comment:
Perhaps add a comment here explaining that this is for turning off telemetry reporting from the controller proxies.